sql/privilege.md

```diff
@@ -324,10 +324,12 @@ auth_spec: {
 
 For more information about the user account, see [TiDB user account management](../sql/user-account-management.md).
 
-- IDENTIFIED BY `auth_string`
+- `IDENTIFIED BY 'auth_string'`
 
-  When you set the login password, `auth_string` is encrypted by TiDB and stored in the `mysql.user` table.
+    It is used to set the login password. `auth_string` is encrypted by TiDB and stored in the `mysql.user` table.
 
-- IDENTIFIED BY PASSWORD `hash_string`
+- `IDENTIFIED BY PASSWORD 'hash_string'`
 
-  When you set the login password, `hash_string` is encrypted by TiDB and stored in the `mysql.user` table. Currently, this is not the same as MySQL.
+    It is also used to set the login password. `hash_string` is a 41-character string similar to `*EBE2869D7542FCE37D1C9BBC724B97BDE54428F1`, which is directly stored in the `mysql.user` table by TiDB. To get this string, use `SELECT password('auth_string')` to encrypt your password.
```

Comment:
41-character -> 41-bit?
@tiancaiamao @morgo

Reply:
@CaitibChen No a bit can only mean a **b**inary dig**it**, it should never be used to describe general strings.

Reply:
@kennytm  Thanks ^_^